### DISCUSSION TITLE
后续能否支持部署到vercel，glitch等平台

### DISCUSSION THREAD

**Discussion by puppet6** [AlistGo/alist#268] [Q&A] [answered]
https://glitch.com
https://vercel.com

**Comment by xhofe** [comment]
暂无此计划 有能力可自行适配，欢迎pr。

**Reply by puppet6** [reply]
库鲁西😭

**Comment by puppet6** [comment]
https://replit.com/
这个嘞，我太菜了呜呜呜

**Reply by ChirmyRam** [reply]
对于replit这个平台，你可以直接把二进制文件放进去跑，但要注意的是这个平台的免费套餐中，所有实例都是强制开源的，你的实例配置完全裸奔能被一键克隆。

**Comment by ChirmyRam** [comment, marked as answer]
vercel、glitch这俩平台是静态网页托管平台，alist是用go语言写的，不能托管上去，要适配这俩几乎是推倒重构。

**Reply by puppet6** [reply]
害，好叭

**Reply by chapterv** [reply]
[看文档支持go的serverless函数](https://vercel.com/docs/concepts/functions/serverless-functions/supported-languages)不知道能不能算支持golang，另外vercel好像支持部署serverless接口，可以挂mysql等。是不是平台更新了，不晓得能不能不用推倒适配vercel呢
